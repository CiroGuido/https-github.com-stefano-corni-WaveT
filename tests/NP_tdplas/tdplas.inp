&tdplas 
which_eps='drl'
eps_0=1000.,
eps_d=1.,
eps_A=0.110224,
eps_gm=0.00757576, 
eps_w0=0.0,
f_vel=0.0,
which_cavity='bui',
xr= 103.93 179.523
yr= 0. 0.
zr= 0. 0.
rr=
94.4858
94.4858
ns=1
/

