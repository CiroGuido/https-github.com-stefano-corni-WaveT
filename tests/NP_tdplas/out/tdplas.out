 Drude-Lorentz dielectric constant
 Number of spheres           2
 I_sphere  X     Y   Z   Radius
     1    103.9300      0.0000      0.0000     94.4858
     2    179.5230      0.0000      0.0000     94.4858
Done reading input, took     0.000s
 GEPOL-GB: COUNTING TESSERAE
 nts_act         364
 GEPOL-GB: GENERATING TESSERAE

 TOTAL NUMBER OF SPHERES=    2
 SPHERE             CENTER  (X,Y,Z) (A)               RADIUS (A)      AREA(A*A)
   1   54.997391514    0.000000000    0.000000000   49.999735736
   2   94.999487315    0.000000000    0.000000000   49.999735736

 TOTAL NUMBER OF TESSERAE=     364
 SURFACE AREA=      43982.55706370(A**2)    CAVITY VOLUME=     820792.97169983 (A**3)
 nts_act         364
 Created output file with surface points
 Matrixes S D have been written out, I'll stop now
